lib/FrontendTool/FrontendTool.cpp

```diff
@@ -1596,14 +1596,14 @@ createDispatchingDiagnosticConsumerIfNeeded(
   // So a value of "4" here means that there would be no heap allocation on a
   // clean build of a module with up to 32 files on an 8-core machine, if the
   // user doesn't customize anything.
-  SmallVector<FileSpecificDiagnosticConsumer::ConsumerPair, 4> subConsumers;
+  SmallVector<FileSpecificDiagnosticConsumer::Subconsumer, 4> subconsumers;
 
   inputsAndOutputs.forEachInputProducingSupplementaryOutput(
       [&](const InputFile &input) -> bool {
-    if (auto subConsumer = maybeCreateSingleConsumer(input))
-      subConsumers.emplace_back(input.file(), std::move(subConsumer));
-    return false;
-  });
+        if (auto subconsumer = maybeCreateConsumerForDiagnosticsFrom(input))
```

Comment:
Comment still relevant here, so copying it in again: "subconsumer" is not the best name for the local variable, now that that refers to something else. (I was fine with the callback name; the type provides the rest of the information.)

Reply:
Thanks. Changed to `consumer`.